platform/engineering/collab-cycle/architecture-intent-template.md

```diff
@@ -0,0 +1,79 @@
+# Architecture Intent Template
+
+__*** DRAFT *** DRAFT *** DRAFT ***__
+
+The Architecture Intent meeting helps your team build a solution that meets Veteran-facing Service Platform (VSP) engineering standards and lowers the potential for launch-blocking issues later in the development cycle.
+
+## What is the purpose of Architecture Intent meeting?
+
+The Architecture Intent meeting is less a formal presentation and more of a discussion.  It not only provides OCTO-DE and Platform with an early understanding of the product/feature your team wants to build, but is also an opportunity to collaborate and provide feedback on the intended implementation and surface any adjustments needed to meet VSP engineering standards. The focus is on making sure your code meets user needs within the constraints of the platform your building on.
+
+## When to schedule an Architecture Intent meeting?
+
+You should schedule an Architecture Intent meeting if any of these apply:
```

Comment:
Is there a size component to this? Like, would a very large, high profile feature, that only reuses current patterns need a review?

Reply:
As above, I'm thinking of two cases: (1) new teams or stuck teams needing help, and (2) major changes.  I will work to make this more explicit in the next draft.